CMakeLists.txt

```diff
@@ -461,7 +461,7 @@ if(BUILD_TESTS)
     NAME recovery_test
     PYTHON_SCRIPT ${CMAKE_SOURCE_DIR}/tests/recovery.py
     CONSENSUS raft
-    ADDITIONAL_ARGS --recovery 2
+    ADDITIONAL_ARGS --recovery 2 --raft-election-timeout 4000
```

Comment:
Why do we need to make this adjustment now?

Reply:
Because a version of the recovery test now kills a node and waits for a new primary to be elected. The default for the raft election timeout is `100000`.